lib/_http_agent.js

```diff
@@ -106,7 +105,8 @@ function Agent(options) {
   });
 }
 
-util.inherits(Agent, EventEmitter);
```

Comment:
Since #24755 was `semver-major`, maybe we should keep using `util.inherits` here and just backport the `debuglog` part ?

Reply:
This definitely depends on #26468 and it's also the reason why the CI fails.

I have no strong opinion about the `util.inherits` part. I guess someone could rely on some checks that would now fail but it is quite unlikely as well. In this case I would just follow @ZYSzys and pull these changes out.